src/Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs

```diff
@@ -52,31 +52,37 @@ public class TestJsonCommand : PSCmdlet
         private JsonSchema _jschema;
 
         /// <summary>
-        /// Process all exceptions in the AggregateException.
-        /// Unwrap TargetInvocationException if any and
-        /// rethrow inner exception without losing the stack trace.
+        /// Prepare a JSON schema.
         /// </summary>
-        /// <param name="e">AggregateException to be unwrapped.</param>
-        /// <returns>Return value is unreachable since we always rethrow.</returns>
-        private static bool UnwrapException(Exception e)
+        protected override void BeginProcessing()
         {
-            if (e.InnerException != null && e is TargetInvocationException)
-            {
-                ExceptionDispatchInfo.Capture(e.InnerException).Throw();
-            }
-            else
+            SchemaRegistry.Global.Fetch = static uri =>
             {
-                ExceptionDispatchInfo.Capture(e).Throw();
-            }
+                try
+                {
+                    string text;
+                    switch (uri.Scheme)
+                    {
+                        case "http":
+                        case "https":
+                            text = new HttpClient().GetStringAsync(uri).Result;
```

Comment:
Should we dispose the HttpClient()? 
Should we cache it for the cmdlet lifetime?

Reply:
I don't know that I'm qualified to answer those questions, but I'm happy to make the changes if someone else can make the decision.

Reply:
Then I suggest to explicitly dispose:
```
using var client = new HttpClient();
text = client.GetStringAsync(uri).Result;
``